src/Fantomas.Tests/UnionTests.fs

```diff
@@ -186,13 +186,11 @@ type CustomerId = CustomerId of int
 [<Test>]
 let ``Single case DU with private access modifier`` () =
    formatSourceString false """
-type CustomerId =
-   private 
-   | CustomerId of int
+type String50 = private String50 of string
    """ config
    |> prepend newline
    |> should equal """
-type CustomerId = private | CustomerId of int
+type String50 = private String50 of string
```

Comment:
Why did you change the test in this case?

Reply:
Used the same case for both tests. Reverted.